.env.example

```diff
@@ -1,7 +1,6 @@
 LOG_LEVEL=debug
 HTTP_PORT=3000
 HTTP_HOST=your.domain.test
-API_VERSION=v2
```

Comment:
How is this value calculated in the new version?

Reply:
It's not calculated at all, but we will need to add it to the corresponding gke-repo as `API_VERSION=v3` so that paths to scripts loaded by the CMP will be correct.